app/templates/account/donate.html

```diff
@@ -0,0 +1,15 @@
+{% extends 'layouts/base.html' %}
+
+{% block content %}
+<div class="ui stackable centered grid container">
+    <div class="twelve wide column">
+        <h2 class="ui header">
+                Donate Now!
```

Comment:
the `Donate Now!` and the `<div>` below should only be indented one in from the `<h2 class="ui header">`